From: Eric S. Raymond <esr@thyrsus.com>
Date: Fri, 17 Dec 2010 07:11:18 -0500
Subject: Apply Charles Curley's fix from <20101009162503.60c5064a@dragon.localdomain>.

He says:

  If you have just rebooted and not otherwise launched gpsd, plugging in a
  gps creates the device file properly, but does not launch gpsd. The
  problem is the test for ${GPSD_SOCKET} in line 35 of
  gpsd.hotplug.wrapper. If you have just rebooted, the socket doesn't
  exist, so the test fails.

  My workaround is to remove the test.

It sucks that udev is such a moving target.
---
 gpsd.hotplug.wrapper |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gpsd.hotplug.wrapper b/gpsd.hotplug.wrapper
index 325fca9..263cae7 100644
--- a/gpsd.hotplug.wrapper
+++ b/gpsd.hotplug.wrapper
@@ -27,7 +27,7 @@ if [ "$ACTION" = "remove" ] ; then
   exit 0
 fi
 
-if [ -S ${GPSD_SOCKET} -a -x /usr/bin/python ]; then
+if [ -x /usr/bin/python ]; then
   exec /lib/udev/gpsd.hotplug "$ACTION" "$DEVNAME"
 fi
 
-- 
